src/e2e/java/teammates/e2e/cases/e2e/AdminSearchPageE2ETest.java

```diff
@@ -0,0 +1,188 @@
+package teammates.e2e.cases.e2e;
+
+import java.time.Instant;
+
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.AccountAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
+import teammates.e2e.pageobjects.AdminHomePage;
+import teammates.e2e.pageobjects.AdminSearchPage;
+import teammates.e2e.pageobjects.AppPage;
+
+/**
+ * SUT: {@link Const.WebPageURIs#ADMIN_SEARCH_PAGE}.
+ */
+public class AdminSearchPageE2ETest extends BaseE2ETestCase {
+    private AdminSearchPage searchPage;
+
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/AdminSearchPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+        putDocuments(testData);
+    }
+
+    @Test
+    public void allTests() {
+        AppUrl url = createUrl(Const.WebPageURIs.ADMIN_SEARCH_PAGE);
+        loginAdminToPage(url, AdminHomePage.class);
+        searchPage = AppPage.getNewPageInstance(browser, url, AdminSearchPage.class);
+
+        browser.waitForPageLoad();
+        StudentAttributes student = testData.students.get("student1InCourse1");
+        AccountAttributes studentAccount = testData.accounts.get("student1InCourse1");
+        InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
+        AccountAttributes instructorAccount = testData.accounts.get("instructor1OfCourse1");
+
+        ______TS("Typical case: Search student keyword");
+        String searchContent = "Student1";
```

Comment:
`Student1` is a rather generic term. This may result in unexpected results when there are other data in the system.